storage/bucket_test.go

```diff
@@ -62,11 +62,12 @@ func TestBucketAttrsToRawBucket(t *testing.T) {
 				ResponseHeaders: []string{"FOO"},
 			},
 		},
-		Encryption:       &BucketEncryption{DefaultKMSKeyName: "key"},
-		Logging:          &BucketLogging{LogBucket: "lb", LogObjectPrefix: "p"},
-		Website:          &BucketWebsite{MainPageSuffix: "mps", NotFoundPage: "404"},
-		Autoclass:        &Autoclass{Enabled: true, TerminalStorageClass: "NEARLINE"},
-		SoftDeletePolicy: &SoftDeletePolicy{RetentionDuration: time.Hour},
+		Encryption:            &BucketEncryption{DefaultKMSKeyName: "key"},
+		Logging:               &BucketLogging{LogBucket: "lb", LogObjectPrefix: "p"},
+		Website:               &BucketWebsite{MainPageSuffix: "mps", NotFoundPage: "404"},
+		Autoclass:             &Autoclass{Enabled: true, TerminalStorageClass: "NEARLINE"},
+		HierarchicalNamespace: &HierarchicalNamespace{Enabled: true},
```

Comment:
Nit: Move HierarchicalNamespace after SoftDeletePolicy for consistency

Reply:
Done.